src/modules/ekf2/EKF/zero_velocity_update.cpp

```diff
@@ -51,7 +51,7 @@ void Ekf::controlZeroVelocityUpdate()
 			Vector3f vel_obs{0, 0, 0};
 			Vector3f innovation = _state.vel - vel_obs;
 
-			const float obs_var = sq(0.001f);
+			const float obs_var = sq(0.2f);
```

Comment:
Could we just do something this?

```suggestion
			const float obs_var = _NED_origin_initialised ? sq(0.2f) : sq(0.001f);
```

I'm mainly thinking of the accel bias preflight with a board sitting untouched.